web/src/features/charts/EmissionChart.tsx

```diff
@@ -32,7 +32,7 @@ function EmissionChart({ timeAverage, datetimes }: EmissionChartProps) {
         data={chartData}
         layerKeys={layerKeys}
         layerFill={layerFill}
-        valueAxisLabel="CO₂eq / min"
+        valueAxisLabel="tCO₂eq"
```

Comment:
I think this should be without `t` prefix, as the unit (t/kg/Pt, etc.) is now shown on the axis tick labels instead.
```suggestion
        valueAxisLabel="CO₂eq"
```